src/Admin/AdminHelper.php

```diff
@@ -211,7 +211,7 @@ public function appendFormFieldElement(AdminInterface $admin, $subject, $element
                 //since doctrine 2.4
                 $modelClassName = $collection->getTypeClass()->getName();
             } elseif ($collection instanceof Collection) {
-                $modelClassName = $this->getEntityClassName($admin, explode('.', preg_replace('#\[\d*?\]#', '', $path)));
+                $modelClassName = $this->getModelClassName($admin, explode('.', preg_replace('#\[\d*?\]#', '', $path)));
```

Comment:
Isn't this a BC-break if someone override `getEntityClassName` ?

Reply:
I've reverted this one, since it's not relevant to this PR